runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/EngineManager.java

```diff
@@ -244,6 +244,21 @@ private void process(
                 binding.vaultId = resolver.resolve(binding.vault);
             }
 
+            if (binding.catalogs != null)
+            {
+                for (CatalogedConfig cataloged : binding.catalogs)
+                {
+                    Pattern pattern = Pattern.compile(cataloged.name);
```

Comment:
This use of Pattern doesn't seem to be able to fail, so not sure why we have it.